### ISSUE TITLE
When I Run the flutter project facing this problem.

### ISSUE THREAD

**Issue by PrasadPanyam** [flutter/flutter#83867]
Launching lib/main.dart on iPhone 12 Pro Max in debug mode...
Running pod install...
Running Xcode build...
Xcode build done.                                           16.4s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **

Xcode's output:
↳
    /Users/sreeram/Desktop/Application/flutter/.pub-cache/hosted/pub.dartlang.org/image_downloader-0.19.2/ios/Classes/SwiftImageDownloaderPlugin.swift:37:17: error: switch must be exhaustive
                    switch status {
                    ^
    /Users/sreeram/Desktop/Application/flutter/.pub-cache/hosted/pub.dartlang.org/image_downloader-0.19.2/ios/Classes/SwiftImageDownloaderPlugin.swift:37:17: note: do you want to add missing cases?
                    switch status {
                    ^
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
    warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99. (in target 'FMDB' from project 'Pods')
    warning: The use of Swift 3 @objc inference in Swift 4 mode is deprecated. Please address deprecated @objc inference warnings, test your code with “Use of deprecated Swift 3 @objc inference” logging enabled, and then disable inference by changing the "Swift 3 @objc Inference" build setting to "Default" for the "Runner" target. (in target 'Runner' from project 'Runner')
    warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99. (in target 'GoogleMaps' from project 'Pods')
    warning: The iOS Simulator deployment target 'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 14.4.99. (in target 'Flutter' from project 'Pods')

Could not build the application for the simulator.
Error launching application on iPhone 12 Pro Max.

**Comment by darshankawar** [comment]
@PrasadPanyam 
Did you try to look / do research as why you are getting the error ? Usually if you google for the error, you will see similar issues with possible solutions.

Check [this](https://stackoverflow.com/questions/64250840/error-when-trying-to-run-or-build-flutter-project-for-ios) SO link and see if any of the solutions mentioned works for you.

**Comment by PrasadPanyam** [comment]
@darshankawar 
I tried all possible solutions but the above problem was not resolved.

**Comment by darshankawar** [comment]
@PrasadPanyam 
Provide `flutter doctor -v` and `flutter run --verbose` logs.
How is this issue different from the one you raised earlier ? https://github.com/flutter/flutter/issues/83771

**Comment by PrasadPanyam** [comment]
flutter doctor -v
[✓] Flutter (Channel stable, 2.2.1, on macOS 11.3.1 20E241 darwin-x64, locale en-US)
    • Flutter version 2.2.1 at /Users/sreeram/Desktop/Application/flutter
    • Framework revision 02c026b03c (7 days ago), 2021-05-27 12:24:44 -0700
    • Engine revision 0fdb562ac8
    • Dart version 2.13.1

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/sreeram/Library/Android/sdk
    • Platform android-30, build-tools 30.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] VS Code (version 1.55.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.22.0

[✓] Connected device (1 available)
    • Chrome (web) • chrome • web-javascript • Google Chrome 91.0.4472.77
    ! Error: Sreeramulu's iPhone is not connected. Xcode will continue when Sreeramulu's iPhone is connected. (code -13)

• No issues found!

-------------------------------
& Flutter run --verbose:
[flutter run --verbose.docx](https://github.com/flutter/flutter/files/6595971/flutter.run.--verbose.docx)

**Comment by darshankawar** [comment]
@PrasadPanyam 
Again, this is same case as with your earlier issue. The verbose log you shared above doesn't contain the actual error you are seeing, but shows below:
```

[  +54 ms] Error: Sreeramulu's iPhone is not connected. Xcode will continue when Sreeramulu's iPhone is connected. (code -13)
[   +9 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[   +3 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[ +144 ms] No supported devices connected.
[ +177 ms] The following devices were found, but are not supported by this project:
                    Chrome (web) • chrome • web-javascript • Google Chrome 91.0.4472.77
                    If you would like your app to run on web, consider running `flutter create .` to generate projects for these platforms.
```
And it failed because there's no device connected, so the above error is expected. Please connect a simulator or physical device and run the command again which will show the actual error for which you logged the issue.

**Comment by PrasadPanyam** [comment]
@darshankawar 
Even I connected the simulator I am getting the same error. Please find the attachments.
![Uploading Screen Shot 2021-06-04 at 12.21.46 PM.png…]()

**Comment by PrasadPanyam** [comment]
<img width="1440" alt="Screen Shot 2021-06-04 at 12 21 58 PM" src="https://user-images.githubusercontent.com/37583509/120758882-018c3500-c530-11eb-910a-aaefcfae7cd8.png">
<img width="1440" alt="Screen Shot 2021-06-04 at 12 21 46 PM" src="https://user-images.githubusercontent.com/37583509/120758934-110b7e00-c530-11eb-870e-31a5010f3a97.png">

**Comment by darshankawar** [comment]
@PrasadPanyam 
Does your project has `ios` folder ? It should have folder for each supported platform, like below for example:

<img width="242" alt="Screenshot 2021-06-04 at 6 17 25 PM" src="https://user-images.githubusercontent.com/67046386/120803417-29e05780-c561-11eb-9f9f-b6822edaa60a.png">

**Comment by darshankawar** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
